<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowStudent</class>
 <widget class="QDialog" name="MainWindowStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="StartTest">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    /* Исходный градиент: от тёмно-фиолетового (#4B0082) к пурпурному (#8B008B) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #4B0082, stop:1 #8B008B);
    color: white;
    border-radius: 20px;
    border: none;
    padding: 10px;
}

QPushButton:hover {
    /* Новый градиент: добавляем ярко-розовый (#FF00FF) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #FF00FF, stop:1 #8B008B);
}

QPushButton:pressed {
    /* При нажатии — более тёмный розовый (#CC00CC) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #CC00CC, stop:1 #6A006A);
}</string>
   </property>
   <property name="text">
    <string>Тест</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="testsTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="labsTable">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Проверочные работы</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Лабораторные работы</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>29</y>
     <width>811</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>811</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Добро пожаловать</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
